learning_assistant/views.py

```diff
@@ -271,3 +288,125 @@ def get(self, request, course_run_id):
         message_history = get_message_history(courserun_key, user, message_count)
         data = MessageSerializer(message_history, many=True).data
         return Response(status=http_status.HTTP_200_OK, data=data)
+
+
+class LearningAssistantSummaryView(APIView):
+    """
+    View to retrieve data about a learner's session with the Learning Assistant.
+
+    This endpoint returns all the data necessary for the Learning Assistant to function, including the following
+    information.
+        * whether the Learning Assistant is enabled
+        * message history information, if the learner is eligible to use the Learning Assistant
+        * audit trial information
+
+    Accepts: [GET]
+
+    Path: /learning_assistant/v1/course_id/{course_run_id}/
```

Comment:
nit: Shouldn't this path include the `/summary` part (or whatever variant you choose for it)?

Reply:
Yes! Thank you for noticing that - very observant 😀.